server/src/main/java/org/opensearch/cluster/action/index/MappingUpdatedAction.java

```diff
@@ -155,19 +155,16 @@ protected void sendUpdateMapping(Index index, Mapping mappingUpdate, ActionListe
         PutMappingRequest putMappingRequest = new PutMappingRequest();
         putMappingRequest.setConcreteIndex(index);
         putMappingRequest.source(mappingUpdate.toString(), XContentType.JSON);
-        putMappingRequest.clusterManagerNodeTimeout(dynamicMappingUpdateTimeout);
+        putMappingRequest.masterNodeTimeout(dynamicMappingUpdateTimeout);
```

Comment:
I don't think you meant to revert this to the deprecated method?

Reply:
I did not